src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -112,24 +114,45 @@ pxp_cfg_t * pxp_get_default_cfg(void)
  *   STATIC FUNCTIONS
  **********************/
 
```

Comment:
Why do you ever need the static `_pxp_zephyr_irq_handler`? You can use directly `PXP_IRQHandler()` for zephyr as well.

Reply:
I can not use it, the `IRQ_CONNECT` macro needs a function with a `(void)(void*)` signature:
[https://docs.zephyrproject.org/apidoc/latest/group__isr__apis.html#ga131739d1faf501a15590053817aba984](https://docs.zephyrproject.org/apidoc/latest/group__isr__apis.html#ga131739d1faf501a15590053817aba984)